Library/Homebrew/cmd/audit.rb

```diff
@@ -714,8 +714,10 @@ def audit_line(line, lineno)
     if @strict
       if line =~ /system (["'][^"' ]*(?:\s[^"' ]*)+["'])/
         bad_system = $1
-        good_system = bad_system.gsub(" ", "\", \"")
-        problem "Use `system #{good_system}` instead of `system #{bad_system}` "
+        unless %w[| < > & ;].any? { |c| bad_system.include? c }
```

Comment:
I guess at this stage it's running on only `system` lines anyway so seems fine to me